packages/block-library/src/column/save.js

```diff
@@ -16,7 +16,10 @@ export default function save( { attributes } ) {
 	} );
 
 	let style;
-	if ( width ) {
+
+	if ( Number.isFinite( width ) ) {
```

Comment:
`width` can be undefined, so we still should keep the check, how about (I included code comment to ensure it doesn't get removed:

```js
if ( width ) {
	// Numbers are handled for backward compatibility when provided with templates.
	style = { flexBasis: Number.isFinite( width ) ? width + '%' : width };
}
```